src/api/goods.ts

```diff
@@ -10,10 +10,11 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => goods.sort((first, second) => (
+      first.name.localeCompare(second.name))).slice(0, 5));
```

Comment:
Keep all chained methods on the same level.
```suggestion
    .then(goods => goods
      .sort((first, second) => (
        first.name.localeCompare(second.name))
      )
      .slice(0, 5));
```